src/Types/BigNum.js

```diff
@@ -13,16 +13,57 @@ exports.zero = lib.BigNum.zero();
 
 exports.one = lib.BigNum.one();
 
+const add = (a, b) => {
```

Comment:
Since you're using an arrow function (which is good, since that's consistent with the rest of our JS code) and the function body is a single line, you can omit the `return` and the braces

```suggestion
const add = (a, b) => a + b;
```